iac-extensions/spring-config/src/main/java/org/sonar/iac/springconfig/tree/impl/SyntaxTokenImpl.java

```diff
@@ -38,6 +39,11 @@ public String value() {
     return value;
   }
 
+  @Override
+  public List<Comment> comments() {
```

Comment:
Why this is needed? We don't attach comments to the SyntaxTokens, AFAIK.

Reply:
Yes, it is needed for the `defaultMetricsVisitor`